Automattic/wp-calypso
# Disconnect: Use feature checks

#### Changes proposed in this Pull Request

* Removes plan slug checks in favor of feature checks
* Updates list of features.

#### Testing instructions



* Navigate to /settings/disconnect-site and select a Jetpack site.
* Click any of the links that are presented.
* Make sure there's a list of up to five features that your site has access to: 
<img width="1510" alt="Screen Shot 2022-05-25 at 2 40 44 PM" src="https://user-images.githubusercontent.com/1398304/170372286-77c13b4e-a7d8-4f87-ab6e-376406ca44ec.png">




Related to p4TIVU-a66-p2
